ASM/c/actor.c

```diff
@@ -83,8 +83,8 @@ void Actor_StoreFlagInRotation(z64_actor_t *actor, z64_game_t *game, uint16_t ac
 
 // For pots/crates/beehives, determine the override and store the chest type in new space in the actor instance
 // So we don't have to hit the override table every frame.
-void Actor_StoreChestType(z64_actor_t* actor, z64_game_t* game) {
```

Comment:
I don't really agree with this, are we conforming to any C standard which says it should be changed this way? To me attaching the * to the type has always made more sense, since it modifies the type to be a pointer

(Not my full review, just took a glance and saw this)

Reply:
There are C standards recommending both ways; see the PR description on why I think placing it with the value makes more sense. Most importantly, we should use a consistent style and the codebase is currently inconsistent on this, though most of the codebase seems to already place it with the value.

Reply:
This one's an age-old debate. I'm primarily a C++ dev, and I'm a strong believer in "pointerness is part of the type, so it goes on the left". 

Yes, it behaves unintuitively when you declare multiple variables on 1 line, because the C standard was imperfect and we're stuck with some of the bad decisions they made. So just don't declare multiple variables on 1 line.
There's enough value in type readability and reinforcing the right mental model to justify sacrificing a slightly more compact niche syntax option, IMO.

Reply:
C may be a weird language but this quirk is internally consistent with the idea of describing types “in reverse”: `int *foo` means “dereferencing `foo` gives an `int`”, hence the use of `*` rather than `&`. Array and function pointer declarations work similarly, e.g. you can't write `int[] bar`. If we're talking about reinforcing mental models, I think it makes sense to use syntax that reflects this idea and is consistent with how variables of those other types are declared.

Reply:
I would argue that it's internally consistent, but counterintuitive. If you have `int *foo`, and you're considering how to interact with foo, you're probably thinking "foo's a pointer to an int`; you reason about foo in terms of its complete type, not in terms of "what's the type I would get if I dereferenced this?". Making that complete type slightly easier to read at a glance seems virtuous.

Also: as you said, this "in reverse" style led to C's function pointer syntax. That seems like a fantastic reason to distance yourself from it as much as possible...

With that said, I also strongly agree with this:

> Most importantly, we should use a consistent style

Reply:
Consulted with some of the other devs, we're in agreement on keeping the asterisk to the left

Reply:
Done. There were a lot of these so I went through it with find and replace. Checked a few files that still look correct, and the code compiles to the same output, so I think we're good.